docs/internal-documentation/maintenance/ReleaseProcess-release.md

```diff
@@ -394,6 +381,36 @@ branch `merge-develop-into-master` is correct, and a pull request into
 
 [(Top)](#table-of-contents)
 
+## Appendix C on how to rollback (revert) after a release
+
+> This should be a seldom operation. This assumes that the remote repository is
+named `origin`.
+
+If by any chance a release needs to be rollbacked, the following command will
+revert to the latest release point referenced by its tag.
+
+```
+git checkout master
+git revert <TAG>
+git push origin master
+```
+
+> Note that the revert command does remove the data but keep the history. See
+[this document][revert documentation] or this [mailing list thread][revert
+question] [and][revert answer 1] [the][revert answer 2] [answers][revert answer
+3] for more about reverting a merge.
+
+Once the problem is fixed in _develop_, prepare the _master_ branch for the
+[merge of _develop_][merge develop into master] by re-reverting the revert
+commit introduced:
+
+```
+git revert <revert commit>
+git push origin master
+```
```

Comment:
I do not see when this will be used in the release process.

Reply:
This appendix has been removed for now.